tests/virtualization/yast_virtualization.pm

```diff
@@ -54,13 +54,13 @@ sub run {
     systemctl 'start libvirtd', timeout => 60;
     wait_screen_change { send_key 'ret' };
     systemctl 'status libvirtd', timeout => 60;
+    send_key 'ret';
```

Comment:
I don't understand what we need this 'ret' for at all. Can you show a job that failed?

Reply:
> Well, taking a look into the latest job in the current Tumbleweed tests seem that your workaround is just fine, see https://openqa.opensuse.org/tests/843910#step/yast_virtualization/45 

ahh, thanks! I tried it on my openQA instance before that job was run and it failed - but that seems to be unrelated, I have some instability problems right now.

> Created a new needle and retriggered the production test https://openqa.opensuse.org/tests/844239

If your new needle looks like this [one](http://agraul-vm.qa.suse.de/tests/99#step/virt_install/27), `virt_install` should pass ;) 

I'll close the PR as the change is not needed.